app/javascript/components/shared_components/forms/controls/FormSelect.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form as BootStrapForm } from 'react-bootstrap';
+import { useController } from 'react-hook-form';
+import Select from '../../utilities/Select';
+
+export default function FormSelect({
+  children, variant, field, defaultValue,
+}) {
+  const { hookForm } = field;
+  const { id: name, validations: rules } = hookForm;
+
+  const {
+    field: { onChange, onBlur },
+    fieldState: { invalid },
+  } = useController({
+    name,
+    rules,
+    defaultValue,
+  });
+
+  return (
+    <BootStrapForm.Group className="mb-2" controlId={field.controlId}>
+      <BootStrapForm.Label className="small mb-0">
+        {field.label}
+      </BootStrapForm.Label>
+      <Select
+        id={field.controlId}
+        onChange={onChange}
+        onBlur={onBlur}
+        defaultOpt={{ title: defaultValue, value: defaultValue }}
```

Comment:
As described in https://github.com/bigbluebutton/greenlight/pull/4143#issuecomment-1319897032.
Currently, the APIs are too optimistic and assumption based.
Because of the complexity of the feature, subsequent PRs will converge towards fixing those limitations in correspondence with the team consensus.